cmd/falco/snippet.go

```diff
@@ -39,6 +40,12 @@ director {{ .Name }} {{ .Type | printtype }} {
 }
 `
 
+func TerraformBackendNameSanitizer(name string) string {
+	invalid := regexp.MustCompile(`\W`)
```

Comment:
nit: `regexp.MustCompile` may cause panic so I suggest to be placed in package global variables in order to avoid runtime panic.

Reply:
@davinci26 How about this?

It's okay to merge but making regex pointer should be placed on package global.

Reply:
Sorry totally forgot about this :(

Reply:
Will push a commit later today